decouple.py

```diff
@@ -138,6 +147,9 @@ def __contains__(self, key):
     def __getitem__(self, key):
         return self.data[key]
 
+    def items(self):
+        return tuple((k, v) for k, v in self.data.items())
```

Comment:
Why not just return self.data.items()?

Reply:
self.data.items() returns list of items, when tuple(...) returns tuple of items. Thought can be replaced with tuple(self.data.items()).